WasatchNET/Logger.cs

```diff
@@ -137,6 +139,18 @@ public void debug(string fmt, params Object[] obj)
                 log(LogLevel.DEBUG, fmt, obj);
         }
 
+        public void header(string fmt, params Object[] obj)

```

Comment:
This is a lifesaver when trying to read through long logs, especially when you've got 8 spectrometers streaming in parallel.